plugins/by-name/aw-watcher/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, ... }:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "aw-watcher";
+  moduleName = "aw_watcher";
+  packPathName = "aw_watcher";
+  package = "aw-watcher-nvim";
+
+  maintainers = [ lib.maintainers.axka ];
+
+  # Optionally, explicitly declare some options. You don't have to.
+  settingsOptions = {
+    bucket = {
+      hostname = defaultNullOpts.mkStr "<hostname of the computer>" ''
+        The hostname to be presented to the ActivityWatch server
+      '';
+
+      name = defaultNullOpts.mkStr "<\"aw-watcher-neovim_\" .. bucket.hostname>" ''
+        The name of the bucket in the ActivityWatch server
+      '';
+    };
+    aw_server = {
+      host = defaultNullOpts.mkStr "127.0.0.1" ''
+        Host to connect to.
+      '';
+
+      port = defaultNullOpts.mkUnsignedInt 5600 ''
+        Port to connect to.
+      '';
+
+      ssl_enable = defaultNullOpts.mkBool false ''
+        Whether the ActivityWatch server should be connected to using HTTPS.
+      '';
+
+      pulsetime = defaultNullOpts.mkUnsignedInt 30 '''';
+    };
+  };
```

```suggestion
  hasSettings = false;
```
It seems like you are trying to package [aw-watcher.nvim](https://github.com/vonPB/aw-watcher.nvim) which has no options.
Am I wrong?

---

Nope, it's this one: https://github.com/lowitea/aw-watcher.nvim

---

Oh my bad, then you can have those settings.